.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "venv/bin/python3"
```

Comment:
is this file needed?

Reply:
ack, no, the vscode ignore hadn't got in yet...I'll remove